src/cljs/athens/electron/db_menu/core.cljs

```diff
@@ -0,0 +1,173 @@
+(ns athens.electron.db-menu.core
+  (:require
+    ["@material-ui/core/Popover" :as Popover]
+    ["@material-ui/icons/AddCircleOutline" :default AddCircleOutline]
+    [athens.db :as db]
+    [athens.electron.db-menu.db-icon :refer [db-icon]]
+    [athens.electron.db-menu.db-list-item :refer [db-list-item]]
+    [athens.electron.dialogs :as dialogs]
+    [athens.style :refer [color DEPTH-SHADOWS]]
+    [athens.views.buttons :refer [button]]
+    [athens.views.dropdown :refer [menu-style menu-separator-style]]
+    [re-frame.core :refer [dispatch subscribe]]
+    [reagent.core :as r]
+    [stylefy.core :as stylefy :refer [use-style]]))
+
+
+;; -------------------------------------------------------------------
+;; --- material ui ---
+
+(def m-popover (r/adapt-react-class (.-default Popover)))
+
+
+;; temporary local defs
+
+;; Save last visited db in local storage and get current-db path from local storage on startup
+(def current-db-path "ec2-3-16-89-123.us-east-2.compute.amazonaws.com")
+
+
+;; make this a subscription handler, and subscribe to this in db-menu
+#_(def dummy-all-dbs
+    [{:name "Athens Test Remote DB"
+      :path "ec2-3-16-89-123.us-east-2.compute.amazonaws.com"
+      :token "x"
+      :is-remote true}
+     {:name "My DB"
+      :path "/Users/coolUser/Documents/athens/index.transit"
+      :is-remote false}
+     {:name "Top Secret"
+      :path "/Users/coolUser/Documents/athens2/index.transit"
+      :is-remote false}])
```

Comment:
Example data when we were using a vector instead of a map. Delete

Reply:
Done.